tsc/OpenTimelineIO_TSC_Charter.md

```diff
@@ -0,0 +1,67 @@
+# Technical Charter (the "Charter") for OpenTimelineIO a Series of LF Projects, LLC
+
+This charter (the "Charter") sets forth the responsibilities and procedures for technical contribution to, and oversight of, the OpenTimelineIO project, which has been established as OpenTimelineIO a Series of LF Projects, LLC (the "Project"). LF Projects, LLC ("LF Projects") is a Delaware series limited liability company. All Contributors to the Project must comply with the terms of this Charter.
+
+## 1. Mission and Scope of the Project
+  * **a.** The mission of the Project is to develop an open source project with the goals indicated in the "README" file within the Project's repositories.
+  * **b.** The scope of the Project includes software development under an OSI-approved open source license supporting the mission, including documentation, testing, integration and the creation of other artifacts that aid the development, deployment, operation or adoption of the open source software project.
+
+## 2. Technical Steering Committee
+  * **a.** The Technical Steering Committee (the "TSC") will be responsible for all technical oversight of the open source Project.
+  * **b.** The TSC voting members are initially the Project's Committers. At the inception of the project, the Committers of the Project will be as set forth within the "CONTRIBUTING" file within the Project's code repository. The TSC may choose an alternative approach for determining the voting members of the TSC, and any such alternative approach will be documented in the CONTRIBUTING file. Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person.
+  * **c.** TSC projects generally will involve Contributors and Committers. The TSC may adopt or modify roles so long as the roles are documented in the CONTRIBUTING file. Unless otherwise documented:
+    * **i.** Contributors include anyone in the technical community that contributes code, documentation, or other technical artifacts to the Project;
+    * **ii.** Committers are Contributors who have earned the ability to modify ("commit") source code, documentation or other technical artifacts in a project's repository; and
```

Comment:
```suggestion
    * **ii.** Committers are Contributors who have been granted the ability to modify ("commit") source code, documentation or other technical artifacts in a project's repository; and
```

Reply:
I agree that "granted" is more clear, in that the TSC must actually grant that role to someone.

Reply:
So it's true that it must be 'granted', but note that the TSC must have non-discriminatory criteria for becoming a committer. 'Earned' is a good word to use to indicate the burden of hitting the requirements is on the contributor and still gives the TSC the ability to vote to approve the new committer. Some projects have more liberal rules, so this wording gives the project more flexibility it's committer policies. Let me know if this helps.

Reply:
I see! My objection to "earned" is that it is not defined anywhere, and is therefore a subjective, not objective word. I snag on subjective words in contracts.... Effectively "earned" is a granting subject to possibly opaque TSC consideration, as opposed to any kind of measurable, non-discriminatory, accumulation of merit which "earn" implies.

Reply:
Yep - does that address your concern?

Reply:
Re-reading this, I see that 2.c.iii. unambiguously explains that Committers are added/removed by majority approval of existing Committers. This addresses the "granted" part in my mind.
With that out of the way, the word "earned" in 2.c.ii. just serves to explain the difference between a Contributor and a Committer, not the mechanism for how to become one. Furthermore, 2.c says "Unless otherwise documented" and we have a whole section on GOVERNANCE.md titled "How to Become a Committer" which covers that.

Reply:
OK - seeing if there's anything open on this still. Sorry to bug ;-)

Reply:
all good, thanks!

Reply:
👍